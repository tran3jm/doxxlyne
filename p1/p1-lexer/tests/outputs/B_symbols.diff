--- outputs/B_symbols.txt	2022-09-14 22:17:09.307757983 -0400
+++ expected/B_symbols.txt	2022-09-14 02:03:18.674642332 -0400
@@ -1,7 +1,4 @@
-SYMBOL   [line 001]  =
-SYMBOL   [line 001]  =
-SYMBOL   [line 002]  <
-SYMBOL   [line 002]  =
-SYMBOL   [line 003]  >
-SYMBOL   [line 003]  =
+SYMBOL   [line 001]  ==
+SYMBOL   [line 002]  <=
+SYMBOL   [line 003]  >=
 SYMBOL   [line 004]  =
