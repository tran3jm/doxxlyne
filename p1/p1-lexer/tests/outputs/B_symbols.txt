SYMBOL   [line 001]  =
SYMBOL   [line 001]  =
SYMBOL   [line 002]  <
SYMBOL   [line 002]  =
SYMBOL   [line 003]  >
SYMBOL   [line 003]  =
SYMBOL   [line 004]  =
