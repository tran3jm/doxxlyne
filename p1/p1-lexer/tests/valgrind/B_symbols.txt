==958853== Memcheck, a memory error detector
==958853== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==958853== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==958853== Command: ../decaf inputs/symbols.decaf
==958853== 
SYMBOL   [line 001]  (
SYMBOL   [line 001]  {
SYMBOL   [line 001]  [
SYMBOL   [line 001]  ]
SYMBOL   [line 001]  }
SYMBOL   [line 001]  )
SYMBOL   [line 001]  ,
SYMBOL   [line 001]  ;
SYMBOL   [line 001]  =
SYMBOL   [line 001]  +
SYMBOL   [line 001]  -
SYMBOL   [line 001]  *
SYMBOL   [line 001]  /
SYMBOL   [line 001]  %
SYMBOL   [line 001]  <
SYMBOL   [line 001]  >
SYMBOL   [line 001]  <=
SYMBOL   [line 001]  >=
SYMBOL   [line 001]  ==
SYMBOL   [line 001]  !=
SYMBOL   [line 001]  &&
SYMBOL   [line 001]  ||
SYMBOL   [line 001]  !
==958853== 
==958853== HEAP SUMMARY:
==958853==     in use at exit: 0 bytes in 0 blocks
==958853==   total heap usage: 2,124 allocs, 2,124 frees, 235,080 bytes allocated
==958853== 
==958853== All heap blocks were freed -- no leaks are possible
==958853== 
==958853== For lists of detected and suppressed errors, rerun with: -s
==958853== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
