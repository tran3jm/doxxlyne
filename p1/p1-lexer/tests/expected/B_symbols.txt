SYMBOL   [line 001]  (
SYMBOL   [line 001]  {
SYMBOL   [line 001]  [
SYMBOL   [line 001]  ]
SYMBOL   [line 001]  }
SYMBOL   [line 001]  )
SYMBOL   [line 001]  ,
SYMBOL   [line 001]  ;
SYMBOL   [line 001]  =
SYMBOL   [line 001]  +
SYMBOL   [line 001]  -
SYMBOL   [line 001]  *
SYMBOL   [line 001]  /
SYMBOL   [line 001]  %
SYMBOL   [line 001]  <
SYMBOL   [line 001]  >
SYMBOL   [line 001]  <=
SYMBOL   [line 001]  >=
SYMBOL   [line 001]  ==
SYMBOL   [line 001]  !=
SYMBOL   [line 001]  &&
SYMBOL   [line 001]  ||
SYMBOL   [line 001]  !
